### ISSUE TITLE
[BUG]: 'Reset zoom' button not working in 7.2.1

### ISSUE THREAD

**Issue by skuske** [highcharts/highcharts-ios#261]
I just found out that the 'Reset zoom' button is no longer working in v7.2.1 - it works fine in 7.2.0, but in 7.2.1 it does not work at all.

This is a critical issue as it prevents us from updating to the latest version.

Sample code:

```
    HIChartView *chartView = [[HIChartView alloc] initWithFrame:self.view.bounds];
    
    chartView = [[HIChartView alloc] initWithFrame:CGRectMake(self.view.bounds.origin.x, self.view.bounds.origin.y + 20, self.view.bounds.size.width, 300.0f)];
    
    HIOptions *options = [[HIOptions alloc]init];
    
    HIChart *chart = [[HIChart alloc]init];
    chart.type = @"column";
    chart.zoomType = @"x";
    options.chart = chart;
    
    HITitle *title = [[HITitle alloc]init];
    title.text = @"Demo chart";
    options.title = title;
    
    HIColumn *series = [[HIColumn alloc]init];
    series.data = @[@49.9, @71.5, @106.4, @129.2, @144, @176, @135.6, @148.5, @216.4, @194.1, @95.6, @54.4];
    options.series = @[series];
    chartView.options = options;
    
    [self.view addSubview:chartView];
```

Zoom in and tap onto the 'reset zoom' button. It does not react to any user interaction.

**Comment by skuske** [comment]
@ihnatmoisieiev 

Can you confirm this? Many thanks.

**Comment by ihnatmoisieiev** [comment]
Hello @skuske,

thank you for the reporting.

I confirm. We are trying to find the issue at the moment.

**Comment by ihnatmoisieiev** [comment]
@skuske the issue has been reported in the main repo: https://github.com/highcharts/highcharts/issues/12438
It's not an HC iOS bug.

**Comment by skuske** [comment]
@ihnatmoisieiev 

Thanks for the update. I hope this get fixed soon...

**Comment by skuske** [comment]
Fixed in 8.0.0
